examples/stamp/stamp.description

```diff
@@ -1,5 +1,6 @@
-// Ballerina supports the `stamp` function which allows to mutate
-// the value itself and set its storage type to a specified type.
-// Stamp function can only be used for `anydata` type variables.
-// Stamp function either returns the value with the stamped type
-// or error.
+// Ballerina supports the `.stamp()` built-in method to change the storage type of a value to a specified type,
```

Comment:
Add some use case type information for why we would need this.

Reply:
Same as above.